nin/dasBoot/LayerManager.js

```diff
@@ -124,6 +136,22 @@ LayerManager.prototype.updateActiveLayersList = function(frame, forceUpdate) {
   }
 };
 
+LayerManager.prototype.showCameraPathVisualizations = function(shouldShow) {
+  for(var i = 0; i < this.layers.length; i++) {
```

Comment:
The linter complains about this code style.
https://github.com/ninjadev/nin#linting